subscriptions/tasks.py

```diff
@@ -298,6 +305,50 @@ def run(self, subscription_id, **kwargs):
 post_send_process = PostSendProcess()
 
 
+class ScheduleDisable(Task):
+
+    """ Task to disable a subscription's schedule
+    """
+    name = "seed_stage_based_messaging.subscriptions.tasks.schedule_disable"
+
+    def scheduler_client(self):
+        return SchedulerApiClient(
+            api_token=settings.SCHEDULER_API_TOKEN,
+            api_url=settings.SCHEDULER_URL)
+
+    def run(self, subscription_id, **kwargs):
+        l = self.get_logger(**kwargs)
+        l.info("Disabling schedule for <%s>" % (subscription_id,))
+        try:
+            subscription = Subscription.objects.get(id=subscription_id)
+            try:
+                schedule_id = subscription.metadata["scheduler_schedule_id"]
+            except:
+                schedule_id = None
```

Comment:
We could put the logging call and `return False` here, then we don't need the if statement below.